src/components/Sidebar/index.jsx

```diff
@@ -105,9 +105,10 @@ class Sidebar extends React.Component {
       // NOTE: keep "Support" link the last nav item
       {
         title: 'Support',
-        to: `${baseUrl}/admin/support`,
+        to: configuration.ENTERPRISE_SUPPORT_URL,
         icon: faLifeRing,
         hidden: !features.SUPPORT,
+        external: true,
```

Comment:
hmm in the test it says external is default false. by default are we going the 'external' route

Reply:
I'm confused here.

Where external is default:
```
IconLink.defaultProps = {
  icon: null,
  isExpanded: false,
  external: false,
```

I'm explicitly only passing it in for the Support item right above your comment. It's otherwise not being appended to any of the sidebar links.

So yes, the default is false as we're not passing it in so it uses the defaultProp value.

Reply:
sorry I think I confused this, default us false. we pass true so looks good!